pkg/pillar/cmd/nim/nim.go

```diff
@@ -671,6 +671,10 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject) in
 					handleLinkChange(&nimCtx)
 					handleInterfaceChange(&nimCtx, ifindex,
 						"LinkChange", true)
+					if isIfNameCrucial(&nimCtx.deviceNetworkContext, change.Attrs().Name) {
+						log.Functionf("Start network connectivity verfication because ifname %s port of DPC at index 0 changed", change.Attrs().Name)
```

Comment:
index 0 in log message is now misleading,right?

Reply:
Addressed as part of a different PR.